subsys/bluetooth/controller/ll_sw/ctrl_internal.h

```diff
@@ -198,6 +184,31 @@ struct connection {
 		} radio_pdu_node_rx;
 	} llcp_terminate;
 
+#if defined(CONFIG_BLUETOOTH_CONTROLLER_CONN_PARAM_REQ)
+	struct {
+		u8_t req;
+		u8_t ack;
+		u8_t state:2;
+#define LLCP_CPR_STATE_REQ      0 /* master / slave send req */
+#define LLCP_CPR_STATE_RSP      1 /* master rej/rsp / slave rej/rsp */
+#define LLCP_CPR_STATE_APP_WAIT 2 /* wait app resp */
+#define LLCP_CPR_STATE_RSP_WAIT 3 /* wait master/slave rsp/rej */
```

Comment:
Just some cosmetic here, I guess you could use an enum for these and then use the enum type. Other than that looks good.